pkg/processor/coin_storage_helper.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Coinbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package processor
+
+import (
+	"context"
+
+	"github.com/coinbase/rosetta-cli/pkg/storage"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+)
+
+var _ storage.CoinStorageHelper = (*CoinStorageHelper)(nil)
+
+// CoinStorageHelper implements the storage.CoinStorageHelper
+// interface.
+type CoinStorageHelper struct {
```

Comment:
Should this file be in storage package if this implements `storage.CoinStorageHelper`?

Reply:
We need to do a big refactor on everything in `processor`. The whole package can likely be removed by aligning helper/handler interfaces with existing package interfaces. We would then just provide an instantiation of the `fetcher` (for example) instead of a `processor` that wraps the `fetcher`.